akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -812,12 +812,18 @@ object ClusterReceptionist {
      */
     class ClientResponseTunnel(client: ActorRef, timeout: FiniteDuration) extends Actor with ActorLogging {
       context.setReceiveTimeout(timeout)
+
+      private val isAsk = client.path.elements.headOption.contains("temp")
```

Comment:
ok, we can add the additional check that the second element starts with "$"
(I prefer to not do full toString)